static/js/show-task.js

```diff
@@ -37,3 +37,21 @@ function saveTask() {
             );
     }});
 }
+
+$(window).on('load', function worker() {
+            search_term = document.getElementById("search_by_task_name_form").search_term.value
```

Comment:
Why not `$("#search_by_task_name_form")`? Also let's be consistent in element ids: either dashes (as in `task-table`) or underscores (`search_by_task_name_form`) as separators.

Reply:
They both are the same but document.getElementById is faster than the jquery way. (Reference link: https://stackoverflow.com/questions/6103766/diference-between-getelementbyid-and-jquery-smth).
And I've made the changes related to the consistency point.